diff --git a/Makefile b/Makefile
index 7c6fa1e4..d1bf80da 100644
--- a/Makefile
+++ b/Makefile
@@ -18,9 +18,9 @@
 
 # read README.external for more information about the configuration
 # batman-adv DebugFS entries:
-export CONFIG_BATMAN_ADV_DEBUGFS=n
+export CONFIG_BATMAN_ADV_DEBUGFS=y
 # B.A.T.M.A.N. debugging:
-export CONFIG_BATMAN_ADV_DEBUG=n
+export CONFIG_BATMAN_ADV_DEBUG=y
 # B.A.T.M.A.N. bridge loop avoidance:
 export CONFIG_BATMAN_ADV_BLA=y
 # B.A.T.M.A.N. distributed ARP table:
